bin/Powheg/production/2017/13TeV/VBF_Z_Z_2l2nu_NNPDF30_nlo_13TeV/VBF_Z_Z_mumununu_nlo_NNPDF30_13TeV.input

```diff
@@ -0,0 +1,97 @@
+! Production parameter for VBF ZZ production 
+vdecaymodeZ1 11  ! PDG code for charged decay product of Z vector boson (11:e;13 mu ...)  
+vdecaymodeZ2 12  ! PDG code for charged decay product of Z vector boson (11:e;13:mu;...)  
```

Comment:
if you want mumu and nu_m nu_m, then you need to use:
vdecaymodeZ1 13
vdecaymodeZ2 14

Reply:
Should be my mistake when copying those files, thanks for noticing this.